R/wwinference.R

```diff
@@ -350,40 +370,41 @@ fit_model <- function(compiled_model,
 #' This function returns a list of MCMC settings to pass to the
 #' `cmdstanr::sample()` function to fit the model. The default settings are
 #' specified for production-level runs, consider adjusting to optimize
-#' for speed while iterating.
+#' for speed while iterating. All input arguments to `cmdstanr::sample()`
+#' are configurable by the user. See `cmdstanr::sample()` documentation
+#' https://mc-stan.org/cmdstanr/reference/model-method-sample.html for a full
+#' description of each argument.
 #'
 #'
 #' @param iter_warmup integer indicating the number of warm-up iterations,
-#' default is `750`
+#' default is `750`.
 #' @param iter_sampling integer indicating the number of sampling iterations,
-#' default is `500`
-#' @param n_chains integer indicating the number of MCMC chains to run, default
-#' is `4`
-#' @param seed set of integers indicating the random seed of the stan sampler,
-#' default is NULL
+#' default is `500`.
+#' @param seed set of integers indicating the random seed of the Stan sampler,
```

Comment:
Revise the description to be more consistent with [`sample()` docs](https://mc-stan.org/cmdstanr/reference/model-method-sample.html). NB:

> The only time seed should be specified as a vector (one element per chain) is if RNG functions are used in transformed data and the goal is to generate different data for each chain.